.github/workflows/pypi.yml

```diff
@@ -21,8 +28,6 @@ jobs:
             WORKDIR: contrib/pyln-testing
           - PACKAGE: pyln-proto
             WORKDIR: contrib/pyln-proto
-          - PACKAGE: pyln-grpc-proto
```

Comment:
Not quite sure why were dropping the pyln-grpc-proto package?

Reply:
We can add it back, but I thought to drop it because:

- It is not included in our `make pyln-release` script, which only specifies `PYLNS=client proto testing` for the three packages we maintain.

- Our documentation also mentions only these three packages for publishing: "This requires keys for each of pyln-client, pyln-proto, and pyln-testing accessible to poetry."

- Additionally, the package on [PyPI](https://pypi.org/project/pyln-grpc-proto/#history) appears to be manually maintained (not with release cycles), with version 24.2.1 released in April and version 0.1.3 following in May.

If we decide to publish it along with the other packages, we can update the script and documentation accordingly.

Reply:
Gotcha, my bad, I must've forgotten to build the release out for that package. I'll do that and then we can re-add it to the pipeline :+1: